### PR TITLE
More descriptive delete/move buttons + red delete

### PR DESCRIPTION
**Pull request by mahval**
Ref: https://github.com/home-assistant/home-assistant-polymer/issues/4030#issue-508172754

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @mahval, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant-polymer#4145).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: src/panels/lovelace/components/hui-card-options.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class HuiCardOptions extends LitElement {
                   "ui.panel.lovelace.editor.edit_card.move"
                 )}</paper-item
               >
-              <paper-item @click="${this._deleteCard}">
+              <paper-item class="delete-item" @click="${this._deleteCard}">
```
Comment (springstan) [inline src/panels/lovelace/components/hui-card-options.ts]: You will have to resolve the build error by replacing `class="delete-item"` with `·class="delete-item"`. Thanks :)

Comment (mahval) [reply src/panels/lovelace/components/hui-card-options.ts]: > You will have to resolve the build error by replacing `class="delete-item"` with `·class="delete-item"`. Thanks :)

Fixed! Thanks for the feedback

Comment (springstan) [reply src/panels/lovelace/components/hui-card-options.ts]: Great! No problem :)

File: src/panels/lovelace/components/hui-card-options.ts [hunk 2]
```diff
@@ -133,6 +133,10 @@ export class HuiCardOptions extends LitElement {
       paper-item {
         cursor: pointer;
       }
+
+      paper-item.delete-item {
+        color: red;
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-card-options.ts]:
```suggestion
        color: var(--google-red-500);
```

File: translations/en.json [hunk 3]
```diff
@@ -1424,8 +1424,8 @@
                         "pick_card": "Which card would you like to add?",
                         "add": "Add Card",
                         "edit": "Edit",
-                        "delete": "Delete",
-                        "move": "Move",
+                        "delete": "Delete Card",
```
Comment (bramkragten) [inline translations/en.json]: Only update `src/translations/en.json`, the other one is maintained by Lokalise

File: src/panels/lovelace/components/hui-card-options.ts (hunk 2 above)
Comment (mahval) [reply src/panels/lovelace/components/hui-card-options.ts]: Thanks for the feedback.

I have changed it to google-red, if this is what you prefer I would recommend changing the other instances of color: red in your project.

**Review by bramkragten — APPROVED** [review]

**Comment by bramkragten** [comment]
Thanks! 🦁

**Pull request merged**